test/fixtures/mysql-ha/main.tf

```diff
@@ -60,6 +60,7 @@ module "mysql" {
   }
 
   // Read replica configurations
+  read_replica_name_suffix                     = "test"
```

Comment:
With the change, let's inject the hyphen here.

Reply:
Done: https://github.com/GoogleCloudPlatform/terraform-google-sql-db/pull/46/commits/173a76b138bd113c9c70e2537b2a09aa17b80973